<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTablesHeaders</class>
 <widget class="QWidget" name="FormTablesHeaders">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="TableWidgetHollow">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="TableWidgetHorizontalHeader">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>0</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="TableWidgetVerticalHeader">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>301</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QTableWidget" name="TableWidgetContent">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>290</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>3</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
